### PR TITLE
datetime.php cleanup

### PR DESCRIPTION
**Pull request by rabuzarus**
remove `function_exists` and added doxygen

for function datetimesel() and function timesel() im not sure of which kind the variables are. I would suppose `string` because all others seem to be a string. But I'm not sure

### CODE REVIEW

File: include/datetime.php [hunk 1]
```diff
@@ -69,15 +95,19 @@ function field_timezone($name='timezone', $label='', $current = 'America/Los_Ang
...
+/**
+ * @brief General purpose date parse/convert function.
+ *
+ * @param string $from Source timezone
+ * @param string $to Dest timezone
+ * @param string $s Some parseable date/time string
+ * @param string $fmt Output format recognised from php's DateTime class
+ *   http://www.php.net/manual/en/datetime.format.php
+ * 
+ * @return string
```
Comment (annando) [inline include/datetime.php]: Could you add a description for the return value?

File: include/datetime.php [hunk 2]
```diff
@@ -300,23 +355,33 @@ function relative_date($posted_date,$format = null) {
...
+ *
+ * A: 5:00 AM Dec 31 San Bruno time. That's precisely when I start celebrating
+ * and become a year older. If you wish me happy birthday on January 1
+ * (San Bruno time), you'll be a day late.
+ *
+ * @param string $dob Date of Birth
+ * @param string $owner_tz (optional) Timezone of the person of interest
+ * @param string $viewer_tz (optional) Timezone of the person viewing
+ * 
+ * @return int
```
Comment (annando) [inline include/datetime.php]: Description?

File: include/datetime.php [hunk 3]
```diff
@@ -52,13 +66,25 @@ function select_timezone($current = 'America/Los_Angeles') {
...
-// 'name', 'label', $value, 'help'
-if (!function_exists('field_timezone')){
+
+/**
+ * @brief Generating a Timezone selector
+ * 
+ * Return a select using 'field_select_raw' template, with timezones
+ * groupped (primarily) by continent
+.* arguments follow convetion as other field_* template array:
+.* 'name', 'label', $value, 'help'
```
Comment (fabrixxm) [inline include/datetime.php]: there is a point before the `*`

Comment (rabuzarus) [inline include/datetime.php]: Ohhh :-/   Thanks. I will correct this in the evening

**Pull request merged**